### PR TITLE
Added several dishes, moved some actions, fixed unrealized dreams & added randomitems

### PR DESCRIPTION
**Pull request by oualid6496**
Moved surprise Jar to randomitems.lua (with correct chances to get items)
Added Shaggy ogre bags + Myserious remains to randomitems.lua
Added new hireling dishes (hirelingfood.lua)
Moved 3 exisiting Jean Pierre dishes to Jeanpierrefood.lua
Added more Jean Pierre dishes to Jeanpierrefood.lua
Fixed Unrealized Dreams

### CODE REVIEW

File: data/actions/scripts/others/consumables/jeanpierrefood.lua [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            doRemoveItem(item.uid, 1)
+            setPlayerStorageValue(cid, config.storage, os.time() + config.exhaust)
+        elseif (item.itemid == 9995) then
+            cid:addCondition(tropicalFriedTerrorbird)
+            doCreatureSay(cid, "Chomp.",TALKTYPE_ORANGE_1)
+            doRemoveItem(item.uid, 1)
+            setPlayerStorageValue(cid, config.storage, os.time() + config.exhaust)
+        elseif (item.itemid == 9996) then
+-- "You don't really know what this did to you, but suddenly you feel very happy."
+-- [TO ADD] Above text (Not sure what colour it has to be)
```
Comment (ghost) [inline data/actions/scripts/others/consumables/jeanpierrefood.lua]: orange color

**Comment by oualid6496** [comment]
Are you sure? Because then you get "Slurp" and that sentence a t the same time, that didn't seem right to me

**Comment by oualid6496** [comment]
Are you sure? Because then you get "Slurp" and that sentence a t the same time, that didn't seem right to me

**Comment by ghost** [comment]
indeed, its actually a sendCancelMessage:

btw you can get the missing food from here:
https://otland.net/threads/hot-cuisine-dishes-for-tfs-1-0.206545/

**Comment by ghost** [comment]
very nice, would be even nicer if you added the change in helmet of the deep instead of giving storage (even better if you edited drowning.lua to not give drown condition to those equipped with 5461(helmet of the deep),12541(helmet of the deep with 300 speed bonus),15408(depth galea))

the change:
to consume the coconut shrimp bake you must have a 5461 in your backpack and it will change into a 12541


but its looking nice so far :D

**Pull request merged**